eng/AfterSigning.targets

```diff
@@ -41,4 +41,35 @@
     <!-- Build a .zip file from each platform's directory and write it to 'artifacts' -->
     <ZipDirectory SourceDirectory="%(LanguageServiceBinary.SourceDir)" DestinationFile="%(LanguageServiceBinary.ZipFile)" />
   </Target>
+
+  <Target Name="_ZipDevKitTelemetryBinaries" AfterTargets="_ZipLanguageServerBinaries" Condition="'$(ArcadeBuildFromSource)' != 'true'">
```

Comment:
We zip the files to publish to the CDN so we can consume the telemetry binaries on the vscode-csharp side.